rails/client/app/bundles/kaiju/components/Project/components/DraggableItem/DraggableItem.scss

```diff
@@ -1,36 +1,29 @@
-.kaiju-DraggableItem {
-  color: #bbb;
-  cursor: grab;
-  display: flex;
-  flex: 0 0 auto;
-  list-style-type: none;
-  padding: 5px 0;
+:local {
+  .description {
+    cursor: pointer;
+    margin-left: auto;
+    margin-right: 5px;
+    visibility: hidden;
+  }
 
-  &:hover {
-    .kaiju-DraggableItem-description {
-      visibility: visible;
-    }
+  .icon {
+    margin-right: 5px;
   }
-}
 
-.kaiju-DraggableItem:hover::before {
-  background-color: var(--kaiju-theme-hover-active, #282828);
-  content: '';
-  height: 24px;
-  left: 0;
-  margin-top: -3px;
-  position: absolute;
-  right: 0;
-}
+  .item {
+    color: #bbb;
+    cursor: grab;
+    display: flex;
+    flex: 0 0 auto;
+    list-style-type: none;
+    padding: 5px 0;
 
-.kaiju-DraggableItem-text {
-  position: relative;
-}
+    &:hover {
+      background-color: var(--kaiju-theme-hover-active, #282828);
```

Comment:
We should also provide the ability to theme the foreground as well.

Reply:
Kaiju currently doesn't have any additional themes. I added these custom properties a while back so we had hooks in place when we were able to start making theming advancements. When we start working on themes we'll definitely assess adding a themeable foreground color